<launch>

  <!-- CON-SAI2 vs CON-SAI2 用の引数 -->
  <!-- 設定されるとconsai2_descriptionのパラメータを上書きする -->
  <arg name="side" default="" />
  <arg name="color" default="" />

  <!-- grSimのロボットを動かす場合はtrue -->
  <arg name="sim" default="false" />

  <group ns="/$(arg color)$(arg side)">

    <!-- ビジュアライザの起動 -->
    <include file="$(find consai2_examples)/launch/visualizer.launch">
      <arg name="side" value="$(arg side)" />
      <arg name="color" value="$(arg color)" />
    </include>

    <include file="$(find consai2_sender)/launch/sender.launch">
      <arg name="load_description" value="false" />
      <arg name="sim" value="$(arg sim)" />
    </include>

    <node name="consai2_control" pkg="consai2_control" type="example_control.py" required="true" output="screen" />

    <node name="game" pkg="consai2_game" type="game.py" required="true" output="screen" />
  </group>
</launch>
